tests/integration/targets/s3_object/tasks/main.yml

```diff
@@ -892,7 +892,7 @@
             - result is changed
             - "'tags' in result"
```

Comment:
There are a bunch of other assert statements that needs to be updated. Are they going to be updated in this PR or an upcoming one?